GPClient/normalloginwindow.cpp

```diff
@@ -50,6 +56,10 @@ void NormalLoginWindow::on_loginButton_clicked()
     const QString username = ui->username->text().trimmed();
     const QString password = ui->password->text().trimmed();
 
+    //save username and password (not secure tho)
```

Comment:
That comment sounds scary.  I worry it might lead some IT departments to prohibit use of this unofficial client.

Can we make it opt-in?

Reply:
Well I haven't looked into `helper::setting`, but I don't really think this saves an hash or crypts the data in any way.

Reply:
In any case, I noticed that in `portalauthenticator.cpp` username and password are saved in the same way, my pr is just auto-completing the fields with something that is already saved on disk by the original application.

Reply:
We can remove the comment if that is the problem :smile:

Reply:
;-)  I'm just looking at the surface here, I'll have to defer to those with actual familiarity with the codebase on the actual security implications.

Reply:
I agree it's not secure to save the credentials in the settings. I'm going to remove it from `portalauthenticator.cpp`. We need to find a better way to store the credentials, something like the credential manager on the system.

Reply:
Will be removed in #111.

Reply:
Is there a standard for that, or is it going to work differently on different DE?

Reply:
Looks like we can save the credentials with: https://github.com/frankosterfeld/qtkeychain/

Reply:
Nice!